TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -63,7 +63,7 @@ Membership in the TOC is determined by a majority vote of the [Steering Committe
 
 While Steering seats are allocated to companies, TOC seats are allocated to individuals, and remain with a person if that person moves to a new employer.
 
-In the event of a vacancy on the TOC, the Steering Committee will solicit candidates for election from the eligible candidates. Eligible candidates are those who have been a [Working Group lead](ROLES.md/#lead) for at least 3 months.
+In the event of a vacancy on the TOC, the Steering Committee will solicit candidates for election.  A strong candidate is one who has made broad, sustained technical contributions over an extended period of time.
```

Comment:
Q - how would the candidate be measured as strong? Prior to this PR, it is based on WG lead and qualifications that met as WG lead.

Reply:
IMO a TOC appointee should be someone that is obvious. Its not the type of role we want to pick someone that we think maybe has started showing signs of _becoming_ a solid TOC member, but a formalization of someone already largely filling the role (i.e. leading broad sustained technical contribution, designs, **reviews**, providing directional feedback on the project level, etc).

TOC is a bit like the supreme court that once elected it is effectively a permanent role. It would be better to leave a vacancy for some time than to prematurely select someone, IMO.

Reply:
I agree with @linsun.  Having vague or interpretive qualifications is just as good as having none.

Reply:
There will be a vote procedure, i do think this is ok to go

Reply:
>A strong candidate is one who has made broad, sustained technical contributions over an extended period of time.
Member

What does extended period of time mean? I agree with @howardjohn here

Reply:
This is a stopgap while we remove working groups (and before revising the TOC concept further). Why don't we just copy the text of the requirements for being a WG lead and add whatever time was appropriate?

Reply:
This is the current criteria for [being a lead on the Working Group](https://github.com/istio/community/blob/master/WORKING-GROUP-PROCESSES.md#leads) is a bit light:

`
Each working group must have 2 or 3 leads, with at least two different organizations represented. Working group leads must meet certain requirements, including having been members of the Istio project for a year, and a maintainer for at least 3 months.
`

We could add those as requirements, but that seems too low a bar, too.  Presumably, the original thought was that a new member of the TOC would have been a successful Working Group Lead for some period of time, and shown some depth and breadth as a contributor.

Ultimately, it's up to the discretion of the Steering Committee to make a final determination.  I think that providing some guidance about an ideal candidate is reasonable, and it's going to be hard to be too descriptive about requirements.

Reply:
I'd agree to improve this bar to at least being active within the latest year. Being too concrete is hard, like how can we say `active`, how can we say `successful `.

Reply:
Given the assumption is the person needs to be in WG lead before TOC, how about increase it to:

Must have been members of the Istio project for 1.5 years, and a maintainer for at least 9 months.

Reply:
If we're going to change the requirements for membership, it's probably best that we discuss this in the next steering committee meeting.  FWIW, given that there is no requirement as to how long a candidate should have been a WG lead before being eligible, I don't think we need to increase the time requirements.

Reply:
You have to have been a maintainer for 3 months to be a WG lead, and a WG lead for 3 months to qualify for TOC.  So that would mean "a maintainer for 6 months".  We could also spell out the membership requirement (15 months) if we think that is necessary.

There's also the implication of the old system having a much smaller pool of WG leads to choose from.  I don't know that we want all applicable maintainers to consider themselves candidates (and potentially feel bad if they are not chosen).

Right now the TOC (with 6 members for life) is representative of the longest serving technical leaders in the project.  It probably should not always be this way, but I am likely to select people based on longevity.  In my mind, the best candidates have been with the project for more like 5 years.

Reply:
I've updated the PR to use the time requirements of the previous criteria, but without the working group lead requirement.